msteams-platform/toolkit/accounts.md

```diff
@@ -53,15 +53,30 @@ You can create for your Visual Studio Professional or Enterprise subscriptions -
 > [!IMPORTANT]
 > During development, you must load your app within your Teams without distributing it. This is known as **sideloading**.
 
-One of the ways to check if you have a Teams account, verify if you can sideload apps in Teams:
+The following are the two different ways to enable sideloading:
```

Comment:
here should be two different way to check/verify whether sideloading permission is enabled. Not to enable sideloading.